src/Processors/QtiV2/Out/QuestionTypes/PlaintextMapper.php

```diff
@@ -13,19 +14,36 @@ public function convert(BaseQuestionType $questionType, $interactionIdentifier,
     {
         /** @var plaintext $question */
         $question = $questionType;
-
+        $questionData = $question->to_array();
+        
+		// $feedbackOptions is used to create feedback or modelfeedback elements in responseProcessing
+        $metadata = $question->get_metadata();
+        $feedbackOptions = [];
+        
+        if (isset($metadata) && !empty($metadata->get_distractor_rationale())) {
+            $feedbackOptions['genral_feedback'] = $metadata->get_distractor_rationale();
+        }
+        
         $interaction = new ExtendedTextInteraction($interactionIdentifier);
         $interaction->setLabel($interactionLabel);
         $interaction->setPrompt($this->convertStimulusForPrompt($question->get_stimulus()));
         $interaction->setFormat(TextFormat::PLAIN);
         $interaction->setMinStrings(1);
-        $interaction->setMaxStrings(1);
+        if(isset($questionData['max_length'])){
```

Comment:
Can we have spaces beside `if condition`?

Reply:
Done